src/components/WhoIsEdit/WhoIsEdit.jsx

```diff
@@ -24,9 +24,8 @@ function WhoIsEdit({ contacts, isOpen, checkAll, onChange, domain }) {
                     }
                     return disclosed_attributes.size;
                 })(),
-            isCheckedAll:
-                domain.registrant.org || disclosed_attributes.size === contactsList.length * 2,
-            totalCount: acc.totalCount + 2,
+            isCheckedAll: disclosed_attributes.size === contactsList.length * 3,
```

Comment:
what differents between size and length?
what number does size start counting from? 0 or 1? :)